packages/near-membrane-base/src/environment.ts

```diff
@@ -205,7 +203,8 @@ export class VirtualEnvironment implements MembraneBroker {
                     };
                 }
             }
-            redDescriptors[key] = redDescriptor;
+            // "as any" supresses the "Type 'symbol' cannot be used as an index type."
+            redDescriptors[key as any] = redDescriptor;
```

Comment:
☝️ Is typescript gonna ever allow PropertyKey things without warning?

Reply:
Lol, I WISH ;)